storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -132,12 +125,10 @@ static bool reopen_log_files() {
   log.m_log_flags = log_flags;
   log.m_log_uuid = log_uuid;
   log.m_files = std::move(files);
-  auto file = log.m_files.front();
-  log_encryption_read(log, file);
+  auto file = log.m_files.find(desired_lsn);
```

Comment:
is the encryption information only in one of the ib_logfile* files?

side question: do we write encryption info in every ib_redo* file?